openreferee_server/app.py

```diff
@@ -26,6 +26,7 @@ def create_app():
     register_error_handlers(app)
     db.init_app(app)
     register_db_cli(app)
+    NotifyExtension(app, os.environ.get('NOTIFY_URL'), os.environ.get('NOTIFY_TOKEN'))
```

Comment:
Use `os.environ[...]` if you expect this to always be present

Reply:
This will throw an exception if the keys are not present, where as get will soft return None

I explicitly handle none, plus as these variables can be figured with flask config instead of environmental variables if that is needed later.

Is there a reason why you prefer to use a dict lookup over get?

Reply:
I generally prefer to fail early: If required config keys are missing, I want it to fail at startup time. That's something I see immediately because the app is clearly broken.

If the missing value leaks into the config and it only fails later when trying to send a request, there's a good chance I won't notice it until a real request from a user's action arrives.

That said, I think it makes sense to simply disable notifications if these settings are missing. I'd do that by not initializing the extension and silently doing nothing later when the extension is not on the app context (based on your current architecture). Without the extension structure I'd simply return early in the function to send a notification if the config values are None,

Reply:
Yeah, fair. I'll refactor this now then.